crud/select/plan.lua

```diff
@@ -231,7 +244,8 @@ function select_plan.new(space, conditions, opts)
     -- get sharding key value
     local sharding_key
     if scan_value ~= nil and (scan_iter == box.index.EQ or scan_iter == box.index.REQ) then
-        sharding_key = extract_sharding_key_from_scan_value(scan_value, scan_index, sharding_index)
+        sharding_key = extract_sharding_key_from_scan_value(scan_value, scan_index, sharding_index,
```

Comment:
sharding_index is always primary key here? Do you need so?

Reply:
as we discussed below I keep `extract_sharding_key_from_scan_value()` without changes as is.